File: app/src/main/java/ru/melodin/fast/api/model/VKGraffiti.java
package ru.melodin.fast.api.model;

import androidx.annotation.NonNull;

import org.json.JSONObject;

import java.io.Serializable;

public class VKGraffiti extends VKModel implements Serializable {

    private static final long serialVersionUID = 1L;

    public String url;
    public int width, height;

    VKGraffiti(@NonNull JSONObject o) {
        this.url = o.optString("url");
        this.width = o.optInt("width");
        this.height = o.optInt("height");
    }
}

File: app/src/main/java/ru/melodin/fast/api/model/VKGraffiti.kt
package ru.melodin.fast.api.model

import org.json.JSONObject

import java.io.Serializable

class VKGraffiti internal constructor(o: JSONObject) : VKModel(), Serializable {

    var url: String = o.optString("url")
    var width: Int = 0
    var height: Int = 0

    init {
        this.width = o.optInt("width")
        this.height = o.optInt("height")
    }

    companion object {

        private const val serialVersionUID = 1L
    }
}
